CHECKOUT_code

```diff
@@ -35,8 +35,8 @@ release="main"
 
 fv3_release=$release
 phy_release=$release
-fms_release="2023.02"
-drivers_release="2023.02"
+fms_release="2023.04"
+drivers_release=$release
```

Comment:
It is good that we default to `main` for our base checkout, although it may be worthwhile to note in the README that this by default checks out the 'bleeding edge' code.